app/services/artemis_service.rb

```diff
@@ -16,11 +16,11 @@ def get_batch(add = 'zone,zone.sub_stage,barcodes,custom_data,seeding_unit,harve
   end
 
   def get_batch_by_id(id, add = 'zone,zone.sub_stage,barcodes,custom_data,seeding_unit,harvest_unit,sub_zone,custom_data.custom_field')
```

Comment:
you sure we don't want this `force` here?

Reply:
I do, updated.